.github/sync.yml

```
@@ -63,5 +63,5 @@ group:
     Qbox-project/safecracker
     Qbox-project/txAdminRecipe
   files:
-  - source: repo-template/.github/
-    dest: .github/
+  - source: repo-template/
+    dest: ./
```

```suggestion
  - source: repo-template/.github/
    dest: .github/
  - source: repo-template/.editorconfig
    dest: .editorconfig
```
I'm not sure how syncing the entire base folder works... that might be messy and lead to unexpected changes, maybe specifying the files separately is better?

---

🤷🏻 shouldn't do anything bad, it just copies the files to the root 🧐

---

Seems like a pointless change tbh, works already, why change it